src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -1,33 +1,42 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Please rename to NgramHashing, since both the estimator and the transformer are on the same file.

Reply:
I will, but in last iteration of this PR.

---
In reply to: [238849479](https://github.com/dotnet/machinelearning/pull/1811#discussion_r238849479) [](ancestors = 238849479)